controllers/node_kernel_reconciler.go

```diff
@@ -3,6 +3,7 @@ package controllers
 import (
 	"context"
 	"fmt"
+        "strings"
```

Comment:
I believe there are too many tabs here.

Reply:
@qbarrand don't we have a linter check running on each PR?